app/javascript/react/src/containers/ShowPage.js

```diff
@@ -0,0 +1,59 @@
+import { Link } from 'react-router';
+import React, {Component} from 'react';
+
+class ShowPage extends Component{
+  constructor(props){
+    super(props)
+    this.state = {
+      celestial: {}
+    }
+  }
+
+  componentDidMount(){
+    let id = this.props.params.id
+    fetch(`http://localhost:3000/api/v1/celestials/${id}`)
+      .then(response => {
+        if (response.ok) {
+          return response;
+        } else {
+          let errorMessage = `${response.status} (${response.statusText})`,
+          error = new Error(errorMessage);
+          throw(error);
+        }
+      })
+      .then(response => response.json())
+      .then(response => {
+        this.setState({ celestial: response})
```

Comment:
Try to keep your spacing in consistent. On line 26 there should be a space after response OR no space between the opening braces and celestial.